Update headers style 569

goldfarb opened the pull request:
[569](https://github.com/COVID19Tracking/website/issues/569)

Updated subhead style

![image](https://user-images.githubusercontent.com/3722037/79478621-a15bcc00-7fd9-11ea-804a-ce4de6a6293f.png)

Updated chart head and subhead style

![image](https://user-images.githubusercontent.com/3722037/79478685-b33d6f00-7fd9-11ea-8435-c9b0b4723e77.png)

Updated notes, and guideline titles

![image](https://user-images.githubusercontent.com/3722037/79478813-dc5dff80-7fd9-11ea-8eb8-e2245a4b8d83.png)

julialedur commented:
awesome! 🎉 can we make the chart titles bold? 

also, this was my bad, but the guideline titles need to be a bit bigger than the body-text

goldfarb commented:
Updated guideline title size

![image](https://user-images.githubusercontent.com/3722037/79481265-3e6c3400-7fdd-11ea-8bcb-1937532426cd.png)

Updated chart title to bold 

![image](https://user-images.githubusercontent.com/3722037/79481292-47f59c00-7fdd-11ea-9b12-4c43f63976b9.png)

jeremiak commented:
I'm confused why this has been pending a check for a long time and then when I go see what "checks" are being run Github can't find any for this commit. So I'm going to merge this since its a straightforward change.
